README.md

```diff
@@ -1 +1,206 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://explore-the-countries.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- PRESENTATION -->
+
+##  Presentation <a name="presentation"></a>
+
+- [Presentation Link ](https://www.loom.com/share/a9c4ba2ebfcc4b77afa04ae6cc02f2b0)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- <a href="https://nodejs.org/en/download">Node.js</a>
+- <a href="https://docs.npmjs.com/downloading-and-installing-node-js-and-npm">npm</a>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+ git clone https://github.com/siddghosh108/React-Country-Capstone.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd countries-capstone-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+### Webpack Development Server
+
+To run the webpack development server, execute the following command:
+
+```bash
+npm start
+```
+
+### To track CSS linter errors, run:
+
+```bash
+npx stylelint "**/*.{css,scss}"
+```
+
+- To track JavaScript linter errors, run:
+
+```bash
+npx eslint "**/*.{js,jsx}"
+```
+
+### Build
+
+To build the project, run the following command:
+
+```bash
+npm run build
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+npm test
+```
+
+### Deployment
+
+- You can deploy it using: Github Pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Siddhartha Ghosh**
+
+- GitHub: [@siddghosh108](https://github.com/siddghosh108)
+- Twitter: [@siddharthaghos9](https://twitter.com/siddharthaghos9)
+- LinkedIn: [@siddhartha-ghosh-65902718](https://www.linkedin.com/in/siddhartha-ghosh-65902718)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Login page**
+- **More details of the countries**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you find this project helpful or interesting, star this repository on GitHub to let me know you like the project. ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Special thanks to **Nelson Sakwa** who has the owner of the design.
+
```

Comment:
- [ ] Hi, I would advise you to provide a link to `Nelson Sakwa` name, which can Navigate to his website. You can do it like this: Special thanks to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) who has ... Because this way looks more appropriate and professional.

Reply:
I made the following change n the readme file:
<a href="https://www.behance.net/sakwadesignstudio">Nelson Sakwa</a>
So now the user will get the link to go to the original website